components/opening_hours_editor.vue

```diff
@@ -37,6 +37,16 @@
       <v-card class="pa-2">
         <template v-if="!selectTime">
           <h3 class="subtitle-1">{{ $t('opening_hours_editor.select_days') }}</h3>
+          <v-checkbox
```

Comment:
Use [switch](https://material.io/develop/web/components/input-controls/switches/) for shortcuts:

```suggestion
          <v-switch
```